File: cpp/estructura_de_datos/data_structures/LinkedList/java/src/Node.java
public class Node {

    private int load;
    private Node next;

    public Node(int l) {
        load = l;
        next = null;
    }

    public int getLoad(){
        return  load;
    }

    public  void setLoad(int l) {
        load = l;
    }

    public  Node getNext() {
        return next;
    }

    public void setNext(Node n) {
        next = n;
    }
}

File: cpp/estructura_de_datos/data_structures/LinkedList/cpp/otros/Node.cpp
#include "Node.h"

Node::Node(int n)
{
	load = n;
	next = 0;
}

void Node::setLoad(int n)
{
	load = n;
}

int Node::getLoad()
{
	return load;
}

void Node::setNext(Node* n)
{
	next = n;
}

Node* Node::getNext()
{
	return next;
}